### PR TITLE
[CI] Bound the CUDA graph capture range in test launches and lift the spec fixture's admission cap

### PR DESCRIPTION
**Pull request by hnyls2002**
Test servers live for one file, so the production-sized capture range is never amortized: on a full CI run, GPU boot is 12% (1-gpu-5090) / 25% (1-gpu-h100) of the pool. `popen_launch_server` now caps the graph buckets (prefill 1024, decode 64 -- 97% of measured CI prefill batches and decode steps fall inside), benchmarks opt out, and the EAGLE fixture admits 64 instead of 8 (gsm8k 24.0s -> 9.9s on a 5090); the three classes whose `bs * spec_tokens * vocab` fp32 logits would OOM stay pinned.

---
### CI States

Latest PR Test (Base): :no_entry_sign: [Run #31063431801](https://github.com/sgl-project/sglang/actions/runs/31063431801)
Latest PR Test (Extra): :x: [Run #31063431640](https://github.com/sgl-project/sglang/actions/runs/31063431640)

### CODE REVIEW

**Comment by hnyls2002** [comment]
/rerun-test test_spec_eagle.py test_spec_eagle_topk.py test_spec_eagle_topk_page.py test_spec_eagle_page.py test_spec_eagle_triton.py

**Comment by hnyls2002** [comment]
/rerun-test test_spec_eagle_fa3.py test_spec_eagle_stress.py test_spec_eagle_parity.py test_spec_standalone.py

**Comment by hnyls2002** [comment]
/rerun-test test_vision_openai_server_a.py test_openai_server.py test_bench_serving_1gpu_part1.py
